.travis.yml

```diff
@@ -3,25 +3,27 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
-  - 0.7
   - 1.0
   - 1.1
+  - nightly
+matrix:
+  allow_failures:
+    - julia: nightly
 cache:
  directories:
    - /home/travis/.julia
 addons:
   apt_packages:
     - gfortran
 before_install:
-  - julia -e '(VERSION >= v"0.7" && using Pkg); Pkg.rm("PowerModels"); Pkg.rm("PowerModels")'
-  - julia -e '(VERSION >= v"0.7" && using Pkg); Pkg.update()'
+  - julia -e 'using Pkg; Pkg.rm("PowerModels"); Pkg.rm("PowerModels")'
+  - julia -e 'using Pkg; Pkg.update()'
```

Comment:
I would remove the `cache`, `addons`, and `before_install` sections (the latter will help nightly pass also I think). Recent testing seems to show that they aren't necessary, particularly for when you add the `Project.toml`

Reply:
Yup.  That's the plan.